### PR TITLE
Adding cdst package

### PR DESCRIPTION
**Pull request by calvinwy**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by calvinwy** [comment]
@conda-forge/core
This is the first time trying to setup my package on conda-forge, and I have tried to follow the step-by-step instruction here, but it seem to fail while building the docker build step and showing the following error:
```
ERROR:/home/conda/staged-recipes-copy/.ci_support/compute_build_graph.py:invalid recipe dir: /home/conda/staged-recipes-copy/recipes/cdst
```
Could you please kindly advise what should I do?  I tried to look up the error on google without success...  thanks in advance.

**Comment by calvinwy** [comment]
@conda-forge/staged-recipes @conda-forge/core

This is the first time trying to setup my package on conda-forge, and I have tried to follow the step-by-step instruction here, but it seem to fail while building the docker build step and showing the following error:
```
ERROR:/home/conda/staged-recipes-copy/.ci_support/compute_build_graph.py:invalid recipe dir: /home/conda/staged-recipes-copy/recipes/cdst
```
Could you please kindly advise what should I do?  I tried to look up the error on google without success...  thanks in advance.

File: recipes/cdst/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
+{% set name = "cdst" %}
+{% set version = "0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+build:
+  noarch: python
```
Comment (carterbox) [inline recipes/cdst/meta.yaml:9]: A package cannot be noarch *and* have compiled modules using cython.

https://conda-forge.org/docs/maintainer/knowledge_base.html?highlight=noarch#noarch-python

However, I'm not sure that this package compiles anything. Why does it depend on cython?

**Review by carterbox — CHANGES_REQUESTED** [review]
It looks like your missing recipe issues were resolved.

The latest error seems to be that you are asking for unavailable packages. All of your dependencies must be available on the conda-forge channel. We list them all [here](https://conda-forge.org/feedstock-outputs/). Some may be under different names than you may expect.

**Pull request closed without merging**